lib/order.rb

```diff
@@ -0,0 +1,37 @@
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :show_menu, :selection, :total
+  def initialize
+    @show_menu = [{"1": "Garlic Bread with cheese", price: 6},
+                  {"2": "Pepperoni", price: 12},
+                  {"3": "Hawaiian", price: 11}
+                  ]
+    @selection = []
+    @total = nil
+  end
+
+  def select_pizza(menu_num, quantity)
+    quantity.times { @selection.push(@show_menu[menu_num.to_i - 1]) }
+  end
+
```

Comment:
- good way to select the pizza by choosing the number on menu, and then using a times block to choose amount 👍